libs/navigation/footer.js

```diff
@@ -11,20 +11,21 @@ const envMap = {
 };
 
 export default async function loadBlock(configs = {}) {
-  const { locale, contentRoot, env = 'prod' } = configs;
+  const { locale, authoringPath, env = 'prod', privacyId, privacyLoadDelay = 3000 } = configs;
   const branch = new URLSearchParams(window.location.search).get('navbranch');
   const miloLibs = branch ? `https://${branch}--milo--adobecom.hlx.page` : envMap[env];
 
   // Relative path can't be used, as the script will run on consumer's app
   const { default: bootstrapBlock } = await import(`${miloLibs}/libs/navigation/bootstrapper.js`);
   const { default: locales } = await import(`${miloLibs}/libs/utils/locales.js`);
   const clientConfig = {
-    contentRoot,
+    privacyId,
```

Comment:
wouldn't we need all the privacy configs ?

Reply:
This is the only required field from configs, rest is already set here.
<img width="753" alt="Screenshot 2024-07-18 at 13 15 22" src="https://github.com/user-attachments/assets/dbe449cf-f03e-4499-80a2-63e50c775309">